src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+import logo from '../../assets/images/logo.png'
+
+const Logo = () => (
+  <img src={logo} className='App-logo' alt='logo' />
```

Comment:
Can you please move styles related to this component under same component folder?
And also avoid className and directly use img in `styles.css`
`src/components/Logo/styles.css`
```
img {
  // all App-logo css goes here
}
```

Reply:
done.
Having CSS on img tag will lead to apply CSS to all img tags available on page.

Reply:
Okay then styled components will solve this problem I guess. Or we should find a solution to avoid conflicts between css of components.
When we give class names for React components, build should create unique class names for every component so no 2 same class names will conflict. Fundamentally class names can be duplicate but not ids. And css for react components should take full advantage of it.
@amarendhar-kybing I remember you  were working on styled components. Can you please check this. 
We will merge this PR, after renaming `App-logo` to `Logo-img` and all similar